### DISCUSSION TITLE
Minimum Max Price option

### DISCUSSION THREAD

**Discussion by jreidel** [erlendsellie/priceanalyzer#14] [Q&A] [answered]
When setting up the analyzer, we have a couple of config options that is not self explaining (and not documented I think):

- Minimum max price for the day before Prizeanalyser Hot Water is active
- Percent difference between Min and Max for the day before bothering 

I just use default (20% and 0.2) but frankly I'm not sure what they do/adjust.

Any hints?

**Comment by erlendsellie** [comment, marked as answer]
Sorry about that. I haven't bothered writing full documentation yet, but it's on my todo-list!
* Minimum max price for the day before Prizeanalyser Hot Water is active
   
Best explained as an example:
 If this is set to 0.2, and the max price for the day is 18 øre (in norway), the Hot water sensor won't be active today.  Nice to have when the spotprice suddenly becomes dirt cheap, and you don't care about optimizing at all.
A config option for the Temperature-sensor was also added in version 1.1. released today, which does the same thing, 

* Percent difference between Min and Max for the day before bothering
This setting will disable the temperature-sensor, if the difference between the max and minimum price is lower than the setting. So if the minimum price is for example 30 øre, and the maximum price is 32 øre, there is not much point in controlling the thermostats based on the spotprice.

Hope this helps!
I'll try to document every config option and every attribute the integration provides soon:-)
